src/css/main.css

```diff
@@ -1358,6 +1358,54 @@ div.leaflet-top > .elevation {
     padding: 0 !important;
 }
 
+/* 
+Batch processor modal
+*/
+.bp-input:required:invalid {

```

Comment:
No changes needed, but just for learning purposes, another way to approach this BP-specific styling could be:

- Give a parent div, such as `<div class="modal-body">`, a BP-related ID like `<div id="bp" class="modal-body">`
- Then in the CSS, use inheritance:

```
#bp .input:required:invalid {
    border: 2px solid #FBEC5D;
}

#bp .select:required:invalid {
    border: 2px solid #FBEC5D;
}
/* etc */
```

Then you wouldn't have to rename all the classes in both the CSS and HTML.